lib/util/cmcd_manager.js

```diff
@@ -496,85 +501,112 @@ shaka.util.CmcdManager = class {
    * @private
    */
   applyResponse_(response, data = {}) {
-    if (!this.config_.enabled) {
-      return;
-    }
-
     const targets = this.config_.targets;
     if (!targets) {
       return;
     }
 
     const responseTargets = targets.filter(
-        (target) => target.mode === 'response',
+        (target) =>
+          target.mode === shaka.util.CmcdManager.CmcdMode.RESPONSE &&
+        target.enabled === true,
+    );
+
+    const allowedKeys = Array.from(new Set([
+      ...shaka.util.CmcdManager.CmcdKeys.V2CommonKeys,
+      ...shaka.util.CmcdManager.CmcdKeys.V2ResponseModeKeys,
+    ]));
+
+    const rawOutput = this.getGenericData_(
+        data,
+        shaka.util.CmcdManager.CmcdMode.RESPONSE,
     );
 
     for (const target of responseTargets) {
-      const output = this.getGenericData_(data);
+      const includeKeys = target.includeKeys || [];
+      const allowedKeysResponseMode = this.checkValidKeys(
+          includeKeys,
+          allowedKeys,
+          shaka.util.CmcdManager.CmcdMode.RESPONSE,
+      );
+
+      const output = this.filterKeys_(rawOutput, allowedKeysResponseMode);
+
       const baseURL = target.url;
+      let request = null;
+      const retryParams =
+          shaka.net.NetworkingEngine.defaultRetryParameters();
 
       if (target.useHeaders) {
         const headers = shaka.util.CmcdManager.toHeaders(output);
         if (!Object.keys(headers).length) {
           return;
         }
 
+        request = shaka.net.NetworkingEngine.makeRequest(
+            [`${baseURL}`], retryParams,
```

Comment:
Esto es mas sencillo asi:

```suggestion
            [baseURL], retryParams,
```